src/makeRobot.js

```diff
@@ -38,7 +38,52 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
```suggestion
  return {
```
you may immediately return the object instead of saving it into a variable and returning right after that